packages/web-components/src/components/progress-indicator/progress-indicator.stories.ts

```diff
@@ -41,83 +46,68 @@ const argTypes = {
 export const Default = {
   args,
   argTypes,
-  render: (args) => {
-    const { iconLabel, secondaryLabelText, spaceEqually, vertical } =
-      args ?? {};
-    return html`
-      <cds-progress-indicator
-        ?vertical="${vertical}"
-        ?space-equally="${spaceEqually}">
-        <cds-progress-step
-          description="${ifNonEmpty(iconLabel)}"
-          label="First step"
-          secondary-label="${ifDefined(secondaryLabelText)}"
-          state="complete"></cds-progress-step>
-        <cds-progress-step
-          description="${ifNonEmpty(iconLabel)}"
-          label="Second step with tooltip"
-          state="current"></cds-progress-step>
-        <cds-progress-step
-          description="${ifNonEmpty(iconLabel)}"
-          label="Third step with tooltip"
-          state="incomplete"></cds-progress-step>
-        <cds-progress-step
-          description="${ifNonEmpty(iconLabel)}"
-          label="Fourth step"
-          secondary-label="Example invalid step"
-          state="invalid"></cds-progress-step>
-        <cds-progress-step
-          disabled
-          description="${ifNonEmpty(iconLabel)}"
-          label="Fifth step"
-          state="incomplete"></cds-progress-step>
-      </cds-progress-indicator>
-    `;
-  },
+  render: ({ secondaryLabel, spaceEqually, vertical, currentIndex }) => html`
+    <cds-progress-indicator
+      ?vertical="${vertical}"
+      ?space-equally="${spaceEqually}"
+      current-index="${currentIndex}">
+      <cds-progress-step
+        description="Step 1: Getting started with Carbon Design System"
+        label="First step"
+        secondary-label="${ifDefined(secondaryLabel)}"
+        complete></cds-progress-step>
+      <cds-progress-step
+        description="Step 2: Getting started with Carbon Design System"
+        label="Second step with tooltip"
+        current></cds-progress-step>
+      <cds-progress-step
+        description="Step 3: Getting started with Carbon Design System"
+        label="Third step with tooltip"></cds-progress-step>
+      <cds-progress-step
+        description="Step 4: Getting started with Carbon Design System"
+        label="Fourth step"
+        secondary-label="Example invalid step"
+        invalid></cds-progress-step>
+      <cds-progress-step
+        disabled
+        description="Step 5: Getting started with Carbon Design System"
+        label="Fifth step"></cds-progress-step>
+    </cds-progress-indicator>
+  `,
 };
 
 export const Interactive = {
   render: () => html`
-    <cds-progress-indicator>
+    <cds-progress-indicator
+      current-index="1"
+      .onChange=${() => alert('Clicked')}>
```

Comment:
This could use a [storybook action arg](https://storybook.js.org/docs/essentials/actions) instead. Not a blocker though.